Documents/start_contribution.md

```diff
@@ -0,0 +1,128 @@
+``` bash
+git clone https://github.com/jdi-testing/jdi-light.git
+```
+If you don't have write access, then first fork the repository to your own GitHub account by clicking the "Fork" button on the repository page. This creates a personal copy of the repository where you can make changes without affecting the original project. Next, clone your forked repository to your local machine using the command:
+
+``` bash
+git clone https://github.com/YOUR_USERNAME/jdi-light.git
+```
+
+Replace YOUR_USERNAME with your GitHub username. This step prepares your local development environment for making contributions.
+
+Insead cloning repo in a terminal you can do that by opening a project from IDE here an examples for different IDEs:
+
+- IntelliJ IDEA:
+    - If you see the Welcome screen, click on "Get from VCS".
+    - If you have already opened a project, select "File" > "New" > "Project from Version Control..." from the main menu.
+- Visual Studio Code:
+    - If you see the Welcome screen, click on "Clone Git repository".
+    - If you have already opened a project, press ctr+shift+G or select Source control in a left menu and push "Clone repository"
+
+
+## 2: Understand the project structure 
+
+Core methods and classes are resides in a jdi-light/jdi-light-core folder
+
+Project contains multiple folders like:
+- **.github/workflows** - in this folder you can find github actions description triggers on merging branches
+- Core methods and classes are resides in a **jdi-light/jdi-light-core** folder
+- **jdi-light-XXXX** - Contains the JDI Light specific module targets specific kind of testing or element base. You can see dependencies betveen modules in pom.xml for each module.
+- **jdi-light-XXXX-tests** - Holds tests for the abovementioned specific module. 
+- **jdi-light-XXXX-tests/src/test/resources** - holds variouce property files for example, the chrome.properties file is used to configure the Chrome driver, and the ff.properties file is used to configure the Firefox driver. 
+
+The test.properties file is a general configuration file that contains various settings for the tests, such as the base URL for the tests, the driver to use, and other settings. Make sure to understand and update these files as necessary for your specific setup.
+for example to run tests in non-headless mode you need configure both common.properties and test.properties:
+- common.properties just comment:
+``` bash
+ #arguments=--headless
+```
+- test.properties set up:
+``` bash
+headless=false
+``` 
+- chrome.properties also have some specific parametres to run browser:
+``` bash
+#arguments=--ignore-certificate-errors --start-maximized --incognito
+#enableVNC=true
+```
+<span style="color:red">**see ➡️ [following](https://jdi-docs.github.io/jdi-light/?java#1-quick-start) to find more about a configuration**</span> 
+
+## 3: Build the Project 
+
+Check your java version - you need atleast 11 java to run it 
```

Comment:
But less then JDK 21! Because of the aspects library

Reply:
added